ci/laravel.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
+    - uses: shivammathur/setup-php@v2
```

Comment:
this change violates the conventions that only github owned actions and github partner actions can non-sha references.